src/execution/mod.rs

```diff
@@ -200,14 +225,14 @@ fn build_production_dataflow<A: Allocate>(
     worker: &mut Worker<A>,
     epoch_config: Py<EpochConfig>,
     resume_epoch: u64,
-    mut step_to_key_to_resume_state_bytes: HashMap<StepId, HashMap<StateKey, StateBytes>>,
+    mut step_to_key_to_resume_state: HashMap<StepId, HashMap<StateKey, State>>,
```

Comment:
Good point: the types are there. I tried to take a pass doing a more semantic rename. I think the big things were to distinguish "resume state" from "current state". Hopefully this helps.

Reply:
I didn't end up using the type aliases, since there are so many variations on storing state that don't compound easily. Let's see how the naming changes feel.